pkg/actors/actors.go

```diff
@@ -1106,7 +1116,11 @@ func (a *actorsRuntime) CreateTimer(ctx context.Context, req *CreateTimerRequest
 					if err != nil {
 						log.Errorf("error invoking timer on actor %s: %s", actorKey, err)
 					}
-				} else {
+					if req.Repeats > 0 {
+						cnt++
+					}
```

Comment:
Do we need to consider the execution result? For example, if there is an error, we  do not increase count.

Reply:
This is a valid point. But consider the case where callback never succeeds. Does that mean we will run the timer indefinitely?

Reply:
Maybe more discussion is needed.

Reply:
cc @halspang @artursouza @yaron2

Reply:
I am OK to invoke n times regardless of error. Each call is not a retry of the previous one, they are independent. If error handling is needed, then retry should be handled independently of this count. For the scope of this issue, the current approach is correct.